### DISCUSSION TITLE
Error setting font family/weight combination

### DISCUSSION THREAD

**Discussion by pmonterolq** [six7/figma-tokens#1951] [Q&A]
Hello! First of all, let me tell you that we've been working with Tokens Studio for a while now and that the whole team loves it. We are very excited that you guys have made our job so much easier and enjoyable. Thank you for all your amazing work!

Now, our little issue. For our tokens, we set the font weight in numbers (400, 700 and 900, instead of normal, bold and black). Also, we added variable fonts to the mix and the parameters arent as precised (like 676 instead of 700 for our variable font LotoQc Roman in bold). This does not impeds our work, but we get this error message : _Error setting font family/weight combination for LotoQc Roman VF/676_.

I wonder if you have plans in the futur to support font weight in numbers, instead of just in descriptive value (bold...).

Thanks again and have a wonderful day!

**Comment by six7** [comment]
Figma currently provides no way to set variable font weights using the plugin, which is why the plugin fails to apply it correctly. Once that changes, we will add support for it 🙏  

You can already set weights by numbers as long as those weights map to something that translates, e.g. `400` or `500` or `600` which the plugin will transform to `Bold`.

**Reply by pmonterolq** [reply]
Thank you so much for taking the time to answer my question :) I hope Figma will accept variable fonts soon. Have a nice day!
